metadata_gui_panel.py

```
@@ -0,0 +1,18 @@
+from PySide6.QtWidgets import QDialog, QVBoxLayout, QLabel, QPushButton
+
+class MetadataDialog(QDialog):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)